alarm/alarm.py

```diff
@@ -0,0 +1,38 @@
+from gmail_handler import send_email
+
+EMAIL_TO_NOTIFY = "rjewing@bu.edu"  # Set to the email you want to notify
+
+
+class Alarm():
+    def __init__(self, threshold_low, threshold_high, name="default_alarm"):
+        self.threshold_low = threshold_low
+        self.threshold_high = threshold_high
+        self.name = name
+
+    def check_threshold(self, value):
+        if value is None:
+            return
+        if not self.threshold_low < value < self.threshold_high:
+            print(
+                f"Alarm {self.name} went out of range ({self.threshold_low}, {self.threshold_high}) with value {value}!")
```

Comment:
Can you provide a patient ID as well; a add to alert database call has to be added here

Reply:
database call will be added once #2 is merged